backstory/pom.xml

```diff
@@ -68,6 +68,13 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
```

Comment:
Could you switch this to mockito-inline so there's just the one dependency?

Reply:
I actually updated this in the fix-pom branch, which is now a part of master. It allows for the inline dependency, but still mainly has core listed in the dependencies list. A simple merge with master should solve this problem.